tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -324,7 +324,7 @@ private static function killAllSessions(): void
         }
 
         $manager = static::createTestManager();
-        $primary = $manager->selectServer(new ReadPreference('primary'));
+        $primary = $manager->selectServer(new ReadPreference(ReadPreference::PRIMARY));
```

Comment:
```suggestion
        $primary = $manager->selectServer();
```

Since PHPC 1.11, [`selectServer()`](https://www.php.net/manual/en/mongodb-driver-manager.selectserver.php#refsect1-mongodb-driver-manager.selectserver-changelog) defaults to a primary read preference. Feel free to refactor any calls like this to simply use no arguments.

Related issue is [PHPC-1937](https://jira.mongodb.org/browse/PHPC-1937). I realize this is unrelated to the PR, so you can consider leaving this as-is and just making these changes in a separate PR if you prefer. Up to you if you'd like to track that in a separate PHPLIB ticket that references PHPC-1937.

Additionally, this may be something you can implement with refactor.

Reply:
The fact that "primary" is the default value is not mentioned in the stub. It makes sense to me now.
I updated many other places in tests.

> Additionally, this may be something you can implement with refactor.

I'm not sure Rector can do that. Search/replace was good enough.

Reply:
> The fact that "primary" is the default value is not mentioned in the stub.

I reckon that's because we'd need to provide a constructed ReadPreference object as the default value, and that's not supported AFAIK. Instead, we just have `null` as a default and some supporting documentation.